### PR TITLE
Add imagehash v4 links

### PR DESCRIPTION
**Pull request by bjlittle**
This PR converts all of the relevant `images/*.png` from old hash filenames to new `imagehash 4.0` hash filenames within the `images/v4` directory.

In order not to unnecessarily bloat the repo, the new hash filenames link to their associated old hash filename.

Note that, the new hash filename `fa81857e857e7a81857e7a817a81857a7a81857e857e7a85857e7a817a81857a.png` has been added to correct a new failing test.

### CODE REVIEW

**Comment by pelson** [comment]
FWIW, you don't need to worry about the git repository size when you have two identical files in a repository: https://stackoverflow.com/a/29949004/741316

It definitely helps for user checkouts though...

File: run_test.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class TestHash(unittest.TestCase):
     def test(self):
         self.maxDiff = None
         exceptions = []
-        for fname in glob.glob("images/*.png"):
+        for fname in glob.glob("images/v4/*.png"):
             phash = imagehash.phash(Image.open(fname), hash_size=_HASH_SIZE)
```
Comment (pelson) [inline run_test.py:19]: Do we have the ability to pass the phash version to the function, or does imagehash not maintain backwards compatible image hashing functionality?

**Review by pelson — APPROVED** [review]
Fine with me.
I don't fully have the bigger picture just yet, but from an incremental change perspective, no objections 👍

File: run_test.py (hunk 1 above)
Comment (bjlittle) [reply run_test.py:19]: It's a breaking change moving to v4. There's no way to generate the old hash from v4, hence the test coverage is only on the new images using the new imagehash v4.

To test the old hashes we'd need to install the pre-v4 imagehash ... and I'm kinda loathed to do that. Is that a safe approach to take?

**Pull request merged**